app/server/__init__.py

```diff
@@ -82,6 +82,33 @@ def create_app():
                     add_transaction_filter(t.address, 'ERC20', None, 'TRANSFER', decimals = t.decimals, block_epoch = config.THIRD_PARTY_SYNC_EPOCH)
     except:
         print('Unable to automatically create filters')
+
+    # If it hasn't been done before, migrate from the old custom attribute scheme to the new one
+    try:
+        with app.app_context():
+            from server.models.custom_attribute import CustomAttribute
+            from server.models.custom_attribute_user_storage import CustomAttributeUserStorage
+            if not db.session.query(CustomAttribute).first():
+                print('Old style custom attributes still present. Migrating to new custom attributes scheme!')
```

Comment:
feels smelly to me to have a migration in the init....

is it possible to get this into alembic migration so that it doesn't need to run every time we boot?
(Acknowleding that alchemy inside a migration is generally not good, but this _might_ be possible to do with old school sql?)